tjpcov/covariance_fourier_gaussian_nmt.py

```diff
@@ -190,7 +193,7 @@ def get_covariance_block(
             fname = "cov_{}_{}_{}_{}.npz".format(*tracer_comb1, *tracer_comb2)
 
         fname = os.path.join(self.io.outdir, fname)
-        if os.path.isfile(fname):
+        if os.path.isfile(fname) and not clobber:
```

Comment:
This will not read the saved covariance block but I think it will recompute it with the same cached workspaces; i.e. with the same binning, scales, etc that had been used for the stored block so the result should be the same.

Reply:
I updated the docstring to indicate it does not erase the cache.  Is that alright?  Also in the tests (test_covariance_fourier_gaussian_nmt line 477 for example), clobber=True and providing a cache will correctly use the cache with clobber on.  Do you suggest any other changes?

Reply:
Sounds good to me